esp/platform/txsummary.cpp

```diff
@@ -28,6 +28,12 @@
 static auto pRequestCount = hpccMetrics::registerCounterMetric("esp.requests.received", "Number of requests received", SMeasureCount);
 #endif
 
+namespace
+{
+    // Iteration of summary entries occurs without knowledge of the summary or its connector.
+    static thread_local CTxOpenTelemetryConnector* otelConnector = nullptr;
```

Comment:
I assume the connector normalizes values/names to Otel standards, is that what we mean by "connector" here?
Also, we're actually interfacing w/ Jtrace rather than otel.

Reply:
The connector has full responsibility for anything needed to normalize names and values for jtrace. I was viewing jtrace as an encapsulation for open telemetry, but if either name here is an issue, should I just make a wholesale change reset back from open telemetry?